src/System.Console/src/System/Console.cs

```diff
@@ -225,7 +225,7 @@ public static int WindowWidth
             {
                 if (value <= 0)
                 {
-                    throw new ArgumentOutOfRangeException(nameof(value), value, SR.ArgumentOutOfRange_NeedPosNum);
+                    throw new ArgumentOutOfRangeException("width", value, SR.ArgumentOutOfRange_NeedPosNum);
```

Comment:
Nit: It's slightly unfortunate that "width" matches neither the argument name "value" nor the property name "WindowWidth", but I agree the small value of matching desktop here and having a semi-descriptive name outweighs the common practice of using "value" and being able to use nameof, so LGTM.